### ISSUE TITLE
npymongo.errors.ServerSelectionTimeoutError: localhost:27017: [Errno 111] Connection refused

### ISSUE THREAD

**Issue by titi-Yu** [liuhuanyong/ProductKnowledgeGraph#2]
python3 collect_info.py时遇到这句话过不去：
self.conn['goodskg']['data'].insert(data)

npymongo.errors.ServerSelectionTimeoutError: localhost:27017: [Errno 111] Connection refused

请问是跟数据库有关吗？

**Comment by frankrenping** [comment]
yes, also i meet same issue, Mr lin, can you share you database for us? thanks

**Comment by paul010** [comment]
这个问题已经解决，原因是pymongo数据库的insert()和insert_one()方法均不能调用“集合”对象
调整插入方式如下跑通，
![image](https://user-images.githubusercontent.com/10044008/149650154-dae9424e-89f5-47b5-948b-503e6b0ab241.png)
